<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>LogViewer</class>
  <widget class="QWidget" name="LogViewer">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>1200</width>
        <height>800</height>
      </rect>
    </property>
    <property name="styleSheet">
      <string notr="true">QWidget#LogViewer {
    background-color: #303030;
}
QLabel {
    color: white;
    font-size: 36pt;
    font-weight: bold;
}
QTextEdit {
    background-color: #404040;
    color: white;
    border: 2px solid #505050;
    border-radius: 10px;
    padding: 15px;
    font-size: 14pt;
    font-family: monospace;
}
QPushButton {
    background-color: #404040;
    color: white;
    border: 2px solid #505050;
    border-radius: 10px;
    padding: 10px;
    font-size: 32pt;
    min-width: 200px;
    min-height: 80px;
}
QPushButton:hover {
    background-color: #505050;
    border-color: #606060;
}
QPushButton:pressed {
    background-color: #606060;
}</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
        <number>20</number>
      </property>
      <property name="leftMargin">
        <number>60</number>
      </property>
      <property name="topMargin">
        <number>60</number>
      </property>
      <property name="rightMargin">
        <number>60</number>
      </property>
      <property name="bottomMargin">
        <number>60</number>
      </property>
      <item>
        <widget class="QLabel" name="titleLabel">
          <property name="text"><string>Log File Viewer</string></property>
          <property name="alignment">
            <set>Qt::AlignCenter</set>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QTextEdit" name="logTextEdit">
          <property name="readOnly"><bool>true</bool></property>
        </widget>
      </item>
      <item>
        <widget class="QPushButton" name="backButton">
          <property name="text"><string>Back</string></property>
        </widget>
      </item>
    </layout>
  </widget>
  <resources/>
  <connections/>
</ui>
